includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -507,7 +533,7 @@ private function getFandomOverview() {
 			'links' => [ ]
 		];
 
-		if ( $this->lang === self::DEFAULT_LANG ) {
+		if ( $this->lang === static::DEFAULT_LANG ) {
 			$out['links'] = [
 				[
 					'type' => 'link-branded',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "link-branded" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)